utilities/calculations.test.js

```diff
@@ -0,0 +1,46 @@
+import data from "../context/mock-data";
+import { calculations } from "./calculations";
+
+describe("calculations", () => {
+  const allActivities = data.allActivities;
+  
+  //   it('should have a get percentages method', () => {
```

Comment:
I left this as a comment because wasn't quite sure how to check if this is a function using jest and was curious if someone knows how.